### DISCUSSION TITLE
HomeKit integration

### DISCUSSION THREAD

**Discussion by VaBanck** [sle118/squeezelite-esp32#249] [Ideas]
Hi, is it possible to integrate into HomeKit?

**Comment by sle118** [comment]
If homekit supports Logitech media server (squeezebox) or AirPlay, then yes. It will be supported natively as it is the responsibility of controlling software to interface with devices and not the opposite.

For example homeassistant does have support for Logitech media server.

**Comment by HomeKidd** [comment]
> If homekit supports Logitech media server (squeezebox) or AirPlay, then yes.

HomeKit does support a Speaker accessory, there are some projects which has already take advantages of AirPlay using [HomeKit](https://github.com/openairplay/airplay2-receiver), hopefully that helps 😄
